benches/src/bin/collect.rs

```diff
@@ -633,6 +633,7 @@ fn linear_regression(x_y: Vec<(u64, u64)>) -> f64 {
 fn dependent_cost(name: &String, x_y: Vec<(u64, u64)>) -> (u64, u64) {
```

Comment:
`x_y` isn't a very helpful name. Would `points` work here? I know we have have a `Point` type below, but I think it would still be an improvement.

If the types are getting confusing you could even set the type as `impl Into<Point>` and create a `From<(u64, u64)>` :P. That's less important than the name.